fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -440,12 +527,14 @@ struct GetMappedRangeWorkload : ApiWorkload {
 		std::cout << "Test configuration: transactionType:" << self->transactionType << " snapshot:" << self->snapshot
 		          << "bad_mapper:" << self->BAD_MAPPER << std::endl;
 
-		Key mapper = getMapper(self);
+		state Key mapper = getMapper(self);
 		// The scanned range cannot be too large to hit get_mapped_key_values_has_more. We have a unit validating the
 		// error is thrown when the range is large.
 		state bool originalStrictlyEnforeByteLimit = SERVER_KNOBS->STRICTLY_ENFORCE_BYTE_LIMIT;
 		(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = deterministicRandom()->coinflip();
 		wait(self->scanMappedRange(cx, 10, 490, mapper, self));
+		int secondsToRun = 30;
```

Comment:
Can you make this a parameter, like `checkStorageQueueSeconds`, in the TOML file?

In the `GetMappedRangeWorkload` constructor, add:
```
checkStorageQueueSeconds = getOption(options, "checkStorageQueueSeconds"_sr, 60.0);
```
so that we can easily control how long this period is.